meilisearch/index.py

```diff
@@ -181,6 +183,35 @@ def get_update_status(self, update_id):
             )
         )
 
+
+    def wait_for_pending_update(self, update_id, timeout_in_ms=2000, interval_in_ms=10):
+        """Wait until MeiliSearch processes an update, and get its status
+
+        Parameters
+        ----------
+        update_id: int
+            identifier of the update to retrieve
+        timeout_in_ms (optional): int
+            time the function should wait before rising a TimeoutError
+        interval_in_ms (optional): int
+            time interval the function should wait (sleep) between requests
+        Returns
+        ----------
+        update: `list`
+            List of all enqueued and processed actions of the index.
+        """
+        start_time = datetime.now()
+        elapsed_time = 0
+        while elapsed_time < timeout_in_ms:
+            get_update = self.get_update_status(update_id)
+            if get_update['status'] != 'enqueued':
+                return get_update
+            sleep(interval_in_ms/1000)
+            time_delta = datetime.now() - start_time
+            elapsed_time = time_delta.seconds * 1000 + time_delta.microseconds / 1000
```

Comment:
What does this line do?

Reply:
It uses the timedelta object to calculate the number of milliseconds since the moment the user called the wait_for_pending_update method. time_delta is an instance of a timedelta object, this just converts it to an int in ms